pkgs/top-level/all-packages.nix

```diff
@@ -16292,6 +16290,7 @@ aliases = with self; rec {
   aircrackng = aircrack-ng; # added 2016-01-14
   arduino_core = arduino-core;  # added 2015-02-04
   asciidocFull = asciidoc-full;  # added 2014-06-22
+  b2 = backblaze-b2; # added 2016-03-06
```

Comment:
If the binary name is changing there isn't really a point of creating an alias. It will just cause more subtle breakage and if we are going to break things we might as well force them to change to the new version.